docs/how-to/gateway-best-practices.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Best practices 
+description: Learn best practices for working with IPFS HTTP Gateways
+---
+
+# Best practices for HTTP Gateways
+
+Various best practices for the use of IPFS gateways are listed below. To learn more about the concepts behind IPFS gateways, including how they work, available providers, types and FAQs, see [IPFS Gateway](../concepts/ipfs-gateway.md). For troubleshooting information, see [Troubleshooting](./gateway-troubleshooting.md).
+
+## When not to use a gateway
```

Comment:
I would move this whole section to the bottom of the page to avoid the best practices starting with when not to use a gateway.

I think this "Delay-sensitive applications" section is questionable. Given that CIDs are immutable, content on IPFS plays really well with HTTP caching. 

If you're publishing a website with IPFS, using a gateway with subdomain resolution works pretty darn well (and fast) if you're loading from the IPFS node providing the content. 

Embedding IPFS into your app doesn't really work for the web/browser-based stuff — for that, we need deeper browser support.